### PR TITLE
Toggle logic

### PR DESCRIPTION
**Pull request by ivergentz**
Dashboard/HomePage ready for the first review including toggle logic for the ride details.

Design is in progress, so please concentrate on reviewing the rest of the code.

### CODE REVIEW

File: cypress/integration/dashboardPage.spec.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-  /// <reference types="Cypress" />
-
   context ('DashboardPage', () => {
     beforeEach(() => {
       cy.visit('http://localhost:3000')
```
Comment (carolinwagner) [inline cypress/integration/dashboardPage.spec.js:3]: You could add ' "baseUrl": "http://localhost:3000/" ' to your cypress.json and then it is simply ' cy.visit('/') ' in your dashboardPage.spec.js

**Review by carolinwagner — APPROVED** [review]
Looks good to me :-) Just added a minor suggestion

File: cypress/integration/dashboardPage.spec.js (hunk 1 above)
Comment (ivergentz) [reply cypress/integration/dashboardPage.spec.js:3]: Thanks. Makes it easier for further tests.

File: src/dashboardPage/RidesListItem.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, {useState} from 'react'
```
Comment (maxbischof) [inline src/dashboardPage/RidesListItem.js:1]: I would give the file "RidesListItem.js" and the component "ListItem" the same name. Maybe just call it Ride or RideItem?

**Pull request merged**